src/components/utils/isReactElement.ts

```diff
@@ -0,0 +1,6 @@
+import { ReactElement } from 'react';
+
+export default (node: unknown): node is ReactElement => {
```

Comment:
Q: Doesn't React.isValidElement already check this ?

Reply:
@eniskraasniqi1 This is basically a type guard, which serves a different purpose than `isValidElement`. I don't think that one can be used in the same way like this one.

Reply:
Yes, you're right. Thanks for the advise! I've updated the PR ;)